### ISSUE TITLE
Note to Debian/ Ubuntu users about kernel.unprivileged_userns_clone

### ISSUE THREAD

**Issue by nvesely** [TheGoddessInari/rambox#106]
Maybe something should be added to the README about adding `kernel.unprivileged_userns_clone=1` to `/etc/sysctl.conf` if you're on Debian/Ubuntu based distros. This setting comes from a custom patchset (see https://lwn.net/Articles/673597/) and is not even in the mainline Linux kernel or applied to any other distros as far as I could find, so it won't be relevant to other Linux users.

**Comment by TheGoddessInari** [comment]
Does it fail without it?

If not, then it's the same restrictions that Chrome has with the feature, except that in Electron it's experimental and there's no apparent way to tell if it's actually running.

**Comment by nvesely** [comment]
Yeah, it fails at launch--you won't even see the GUI. Probably a good sign the sandbox is actually working, at least for Linux :)

I actually found you need to set this on a Brave thread from back when they were using (a fork of) Electron and not building on Chromium proper.
